public/app/features/commandPalette/actions/extensionActions.ts

```diff
@@ -0,0 +1,22 @@
+import { PluginExtensionCommandPaletteContext, PluginExtensionPoints } from '@grafana/data';
+import { getPluginLinkExtensions } from '@grafana/runtime';
+
+import { CommandPaletteAction } from '../types';
+import { EXTENSIONS_PRIORITY } from '../values';
+
+export default function getExtensionActions(): CommandPaletteAction[] {
+  const context: PluginExtensionCommandPaletteContext = {};
+  const { extensions } = getPluginLinkExtensions({
+    extensionPointId: PluginExtensionPoints.CommandPalette,
+    context,
+    limitPerPlugin: 3,
```

Comment:
I think it's good for now, we might want to revisit this in the future (maybe we can allow a higher limit for plugins here as this list can be searched).